lighthouse-core/config/config.js

```diff
@@ -340,27 +339,50 @@ class Config {
     return config;
   }
 
- /**
-  * Return IDs of top-level aggregations from a config. Used by tools driving lighthouse-core
-  * @param {{aggregations: !Array<{name: string}>}} Lighthouse config object.
-  * @return {!Array<string>}  Name values of aggregations within
-  */
-  static getAggregations(config) {
-    return config.aggregations.map(agg => ({
-      name: agg.name,
-      id: agg.id
-    }));
+  /**
+   * @param {!Object<{audits: !Array<{id: string}>}>} categories
```

Comment:
> you mean all this time you've misled us!?

I know! Also, it's dumb that it doesn't default to `string` cause that's what you get no matter what you use anyways